db/knexfile.js

```diff
@@ -1,19 +1,26 @@
 "use strict";
 
+
+const { parse } = require("pg-connection-string");
+
 const AppConstants = require("../app-constants");
+const connectionObj = parse(AppConstants.DATABASE_URL);
+if (["dev", "heroku"].includes(AppConstants.NODE_ENV)) {
```

Comment:
I think it should just be `if (["heroku"]` - when I keep dev in there, I get an error locally that says SSL is not supported.